01-JS-DOM-APIs/Practice1-1/Script/script.js

```diff
@@ -0,0 +1,43 @@
+function load () {
+  console.log('load'); //Only to know the page is loaded
+  document.getElementById("changeclass").style.transition = "all 3s ease-in-out";
```

Comment:
In order to avoid duplicated code, you could store document.getElementById("changeclass") in a let